public/manifest.json

```diff
@@ -39,5 +39,6 @@
   },
   "web_accessible_resources": [
     "oasis-xu-frame.html"
-    ]
+  ],
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAybK17ag+0PwHWbYEE36o+SmN35mcXj3RoM8ZBIlOjetdV3uqG8vdzv0YZ2Td21Y0il+1IKt/d9kslRT67C/S2fZ3XKAf96o+j+9/mgdOBwkL7m8XeALzBCYI3sSPnHf36iZu0w9FM9enUFHmq4HPNsT7BqTTEkdtDRRU4JHb+GGLjWvpyB1WIuv7UIJu64zUcZXsGVWZojRG2eISBJZxaItTuFS1210PebsS85++eeHLPWwMMp7d3UXOZpDYoyS07orZJd4c54hZaMNCKKZ5zbupwEuDvSANvEiuxpXe+hHLRnxOlpmhnj+XqkMBTgTFC57b75vhnvOCNbZ/ylW9xwIDAQAB"
```

Comment:
oh cool

https://developer.chrome.com/docs/extensions/mv3/manifest/key/

Reply:
This results in the same ID. If it was different for development, it won't be as easy to test with any dapps.

Are you worried about security?

Reply:
yeah, the security of experimental code having access to the same storage as the real one. actually is that the case? they'll be on the same origin, so I'd guess that they'll have access to the same storage. or does chrome enforce that you fully wipe one before you can install the other?

Reply:
Just tried:
- if you install Chrome web store first, then you can load development version without removing, and storage is kept
- if you go from development to Chrome web store, then that only shows "Remove" button

Reply:
aw darn that's like the opposite of what I wanted :laughing: 

how about this:

1. we enter a key to get a fixed dev ID here, separate from what the published extension uses
2. we put this public key from the published extension in the documentation
3. dapps should support other wallets anyway :crossed_fingers:

Reply:
Is this worth generating different manifest.json with javascript based on env :confounded:

Reply:
imo no. I'd rather edit a file than set an env var

Reply:
I mean env just as the difference between `dev` and `build`

Reply:
probably not that either, if it were up to me. if someone needed to experiment with new changes and to interact with dapps, they'd perhaps use the real public key in a dev env. or if we need to share around a production build but not clobber the data from people's installation of the published version, we'd do it the other way around.